blocks/inspector-controls/index.js

```diff
@@ -23,11 +27,33 @@ export default function InspectorControls( { children } ) {
 	);
 }
 
-InspectorControls.BaseControl = BaseControl;
-InspectorControls.CheckboxControl = CheckboxControl;
-InspectorControls.RadioControl = RadioControl;
-InspectorControls.RangeControl = RangeControl;
-InspectorControls.SelectControl = SelectControl;
-InspectorControls.TextControl = TextControl;
-InspectorControls.TextareaControl = TextareaControl;
-InspectorControls.ToggleControl = ToggleControl;
+const withDeprecation = ( componentName ) => ( OriginalComponent ) => {
+	class WrappedComponent extends Component {
+		componentDidMount() {
+			// eslint-disable-next-line no-console
+			console.warn( `wp.blocks.InspectorControls.${ componentName } is deprecated use wp.components.${ componentName }` );
```

Comment:
Maybe `wp.blocks.InspectorControls.${ componentName } is deprecated, use wp.components.${ componentName }.`

Added punctuation.